components/HomePage/Testimonials.js

```diff
@@ -66,8 +65,11 @@ const Testimonials = ({ className }) => {
 
   return (
     <ContentSection className={className}>
-      <Grid columns={12}>
-        <Cell width={5}>
+      <Grid columns="repeat(auto-fit,minmax(320px,1fr))" style={{direction: "rtl"}}>
```

Comment:
We have some predefined breakpoints set in `breakpoint.js`, could we use those to set this size here?